src/main/java/com/denizenscript/depenizen/bukkit/events/bigdoors/BigDoorsDoorTogglesScriptEvent.java

```diff
@@ -0,0 +1,96 @@
+package com.denizenscript.depenizen.bukkit.events.bigdoors;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.depenizen.bukkit.objects.bigdoors.BigDoorsDoorTag;
+import nl.pim16aap2.bigDoors.Door;
+import nl.pim16aap2.bigDoors.events.DoorEventToggle;
+import nl.pim16aap2.bigDoors.events.DoorEventTogglePrepare;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class BigDoorsDoorTogglesScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // bigdoors door [toggles|opens|closes]
+    //
+    // @Regex ^on bigdoors door (toggles|opens|closes)$
+    //
+    // @Cancellable true
+    //
+    // @Triggers when a Big Doors door toggles, opens, or closes.
+    //
+    // @Context
+    // <context.door> returns the Big Doors door being toggled.
+    // <context.state> returns the Big Doors open state.
+    //
+    // @Plugin Depenizen, Big Doors
+    //
+    // @Switch door:<id> to only process the event if the door matches the input.
+    //
+    // @Group Depenizen
+    //
+    // -->
+
+    public BigDoorsDoorTogglesScriptEvent() {
+        instance = this;
+    }
+
+    public static BigDoorsDoorTogglesScriptEvent instance;
+    public DoorEventTogglePrepare event;
+    public Door door;
+    public DoorEventToggle.ToggleType toggleType;
+
+    @Override
+    public boolean couldMatch(ScriptPath path) {
+        return path.eventLower.startsWith("bigdoors door");
```

Comment:
Please match the format of couldMatch methods in Denizen... the only value after `return` should be `false` or `true`...